ignition-server/controllers/tokensecret_controller.go

```diff
@@ -118,6 +120,9 @@ func (r *TokenSecretReconciler) SetupWithManager(ctx context.Context, mgr ctrl.M
 	log.Info("SetupWithManager", "ns", os.Getenv("MY_NAMESPACE"))
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&corev1.Secret{}).WithEventFilter(tokenSecretAnnotationPredicate(ctx)).
+		WithOptions(controller.Options{
```

Comment:
currently this is using the default rate limiter:
https://github.com/kubernetes-sigs/controller-runtime/blob/e42a8c17945360813e64bde2c4ff742515b750ba/pkg/controller/controller.go#L133-L134

https://github.com/kubernetes/client-go/blob/4fc3881b7ffe4a5c1c7c3cffc7dd9d9667989730/util/workqueue/default_rate_limiters.go#L37-L45

How is this better than default?

Reply:
I have to see the bucket one…. 10 QPS seems like too much